### PR TITLE
Web cam+ai

### PR DESCRIPTION
**Pull request by KipasaKii**
The basics of camera AI so far

### CODE REVIEW

File: source/main.py [hunk 1]
```diff
@@ -1,3 +1,130 @@
...
+# This is for processing while using webcam https://wellsr.com/python/object-detection-from-webcams-with-yolo/
+# This code contains code for activating webcam. Everything commented out is code for object class detection
+# Yet to be added processing for the color, the object size size..
+
+from imageai.Detection import ObjectDetection # For AI object detection AI
+import cv2 # library for the webcam ( this one probably can also find color)
+import numpy as np 
+import mido
+from mido import Message
+import time
```
Comment (Maucee) [inline source/main.py:11]: Do you make use of the `import time`?

**Review by Maucee — APPROVED** [review]
Code looks good

File: source/main.py [hunk 2]
```diff
@@ -1,3 +1,130 @@
...
+        return 'green'
+    elif blue_pixels > yellow_pixels and blue_pixels > orange_pixels and blue_pixels > green_pixels and blue_pixels > violet_pixels and blue_pixels > red_pixels:
+        return 'blue'
+    elif violet_pixels > yellow_pixels and violet_pixels > orange_pixels and violet_pixels > green_pixels and violet_pixels > blue_pixels and violet_pixels > red_pixels:
+        return 'violet'
+    else:
+        return 'red'
+
+
 if __name__ == "__main__":
```
Comment (Holandsoest) [inline source/main.py:67]: It appears you still have to do a `git merge origin/develop` in your terminal.

**Review by Holandsoest — CHANGES_REQUESTED** [review]
@KipasaKii,  
I really like the demo, but you still need to address my concerns before it is clear to merge.

**Pull request closed without merging**